test/Elastic.Apm.Tests/BackendCommTests/CentralConfig/CentralConfigFetcherTests.cs

```diff
@@ -23,6 +35,101 @@ public class CentralConfigFetcherTests : LoggingTestBase
 	{
 		public CentralConfigFetcherTests(ITestOutputHelper xUnitOutputHelper) : base(xUnitOutputHelper) { }
 
+		[Fact]
+		public void Should_Not_Update_Logger_That_Is_ILogLevelSwitchable()
```

Comment:
Shouldn't this be called `Should_Update_Logger_That_Is_ILogLevelSwitchable`? (and not `Should_Not_Update...`)

Reply:
yep, it should, thanks! (must have introduced when applying consistent naming 😄 )